src/plugins/wizard/public/application/components/top_nav.tsx

```diff
@@ -24,6 +26,25 @@ export const TopNav = () => {
   const config = useMemo(() => getTopNavconfig(services), [services]);
   const indexPattern = useIndexPattern();
 
+  useEffect(() => {
```

Comment:
You don't need a useEffect here. If you don't want to recalculate the value each pretender, just wrap it in a useMemo

Reply:
Speaking to @CPTNB Looks like `chrome.setState` updates state in another component, so a useMemo is not ideal here. Resolving this comment.